README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying Kubernetes/OpenShift cluster
+
+A Kafka/Zookeeper cluster can be deployed on a Kubernete or OpenShift cluster. So, first of all, such deployment is needed.
```

Comment:
I'm not a native speaker ... but the *such deployment is needed* sounds strange. Wouldn't *cluster* be a better word than *deployment*? But it is just my feeling.

Reply:
Agree.